### ISSUE TITLE
prediction with CI not working for burrIII3 fit

### ISSUE THREAD

**Issue by JenniGadd** [bcgov/ssdtools#112]
## The Bug

I am fitting a BurrIII3 fit to some data and when I used the predict function to store my fitted line (so that I can plot it later, using ssd_plot, with a ribbon) 

## A Reprex

```r
mydata <- as.data.frame(c( 4.0, 5.0, 9.0, 24.0, 23.8 , 35.0,  36.0, 54,  62,   64, 64,  84,  143,  153, 230, 1070, 2080))
colnames(mydata) <- "Conc"
Burrdists <- ssd_fit_dists(mydata, dists = c("burrIII3"))
autoplot(Burrdists)

pred <- predict(Burrdists)   ## This works
pred <- predict(Burrdists, ci=TRUE)  ## This doesn't work with the Burr distribution 
```

**Comment by joethorley** [comment]
Thanks @JenniGadd - I am able to reproduce. I suspect its because the Burr has convergence issues but I will look into it.

**Comment by joethorley** [comment]
Fixed as of developmental version V0.2.0.9002
